pkg/action/upgrade.go

```diff
@@ -406,15 +406,15 @@ func (u *Upgrade) failRelease(rel *release.Release, created kube.ResourceList, e
 
 		releaseutil.Reverse(filteredHistory, releaseutil.SortByRevision)
 
-		rollin := NewRollback(u.cfg)
-		rollin.Version = filteredHistory[0].Version
-		rollin.Wait = true
-		rollin.WaitForJobs = u.WaitForJobs
-		rollin.DisableHooks = u.DisableHooks
-		rollin.Recreate = u.Recreate
-		rollin.Force = u.Force
-		rollin.Timeout = u.Timeout
-		if rollErr := rollin.Run(rel.Name); rollErr != nil {
+		rolling := NewRollback(u.cfg)
```

Comment:
this was intentional.

https://www.youtube.com/watch?v=YCGeeCIzBs4

Reply:
Ah. Yeah, I'm familiar with that Americana.